corehq/apps/user_importer/helpers.py

```diff
@@ -229,6 +239,18 @@ def update_locations(self, location_codes, domain_info):
                 user_updated_primary_location_name = get_user_primary_location_name(self.user, self.user_domain)
                 self.logger.add_info(f"Primary location: {user_updated_primary_location_name}")
 
+    def _log_phone_number_changes(self, old_phone_numbers, new_phone_numbers):
+        (items_added, items_removed) = find_differences_in_list(
+            list_to_compare=new_phone_numbers,
+            reference_list=old_phone_numbers
+        )
+
+        for number in items_added:
+            self.logger.add_change_message(_(f"Added phone number {number}"))
```

Comment:
A small change to how we were doing this earlier
https://github.com/dimagi/commcare-hq/pull/30064

Please don't translate message.